content/data/moon/kaguyatc_dtms.md

```diff
@@ -0,0 +1,124 @@
+- Hillshade, as a Cloud Optimized GeoTIFF (COG)
+- An ASP generated intersection error raster showing the triangulation error in the point cloud, as a Cloud Optimized GeoTIFF (COG)
+- Orthoimage using the left observation from the stereopair, provided at the resolution of the DTM, as a Cloud Optimized GeoTIFF (COG)
+- LOLA comma separated value (CSV) file with latitude, longitude, and radius (meters) that was used for the alignment of the point cloud to LOLA
+- An ASP generated quality assurance CSV, the output from the geo_diff tool, showing the point-by-point offsets computed, after alignment, between the ASP generated point cloud and LOLA
+- A Provenance file that tracks the processing done during the creation of the product
+
+### Accuracy, Errors, and Issues
+The horizontal accuracy issues present in the [Kaguya TC](kaguyatc.md) observations caused by uncertainties in the ephemerides are largely corrected in this data set. The alignment of the DTMs to LOLA has absolutely adjusted the horizontal and vertical positions of the DTMs and the DTM-derived orthoimages. The quality assurance descriptive statistics describe the alignment to LOLA though users are cautioned the alignment does a terrific job mean centering the error near zero. Therefore, the latitude, longitude, and offset error points in the QA metrics file should be visualized in a GIS to see the spatial distribution of error and to assess whether a sub-region within the DEM is appropriate for the given use case.
+
+As described above the DTMs are aligned to LOLA. Post alignment, the ASP `geodiff` tool was used to compute the absolute offsets to the provided LOLA point cloud. The estimated mean vertical offset to LOLA at the 99th percentile is 11 centimeters and the median vertical offset at the 99th percentile is 4 centimeters. We do see large potential blunders in a subset of DTMs with a maximum mean vertical offset of 7000 meters and a minimum vertical offset of -1811 meters. For these reasons, users of these DTMs should visualize the error metric files in a GIS.
+
+Next, several DTM mosaics, like the one shown at the top of this document, were created. Each DTM was spot-checked the vertical precision of these DTMs where they overlap, checking for agreement in the height above the lunar sphere. There is an average 2-5 meter offset between DTMs which is assessed to be within the expected error budget (i.e., resolution at higher precision using the techniques employed is not feasible).
+
+Finally, averaged observation mosaics, at the mean 26 meters per pixel resolution, were created using the derived orthoimages. The apriori and final position of the orthoimages was assessed to determine the accuracy of the horizontal alignment. Orthoimages (and therefore DTMs) are assessed to have a horizontal precision inside an orthoimage pixel. In other words, these data are in horizontal agreement to better than 26 meters. See the figure below.
```

Comment:
I may be reading it wrong but wanted to confirm that this part makes sense: "... these data are in horizontal agreement _to better_ than 26 meters." Should it be _to be better_?

Reply:
I do not think so. This is trying to express that the maximum horizontal error is less than 26 meters. The sentence is rewritten in that form now. Does this read better?

Reply:
Yes 👍